src/xwayland.c

```diff
@@ -979,6 +991,27 @@ handle_xwm_ready(struct wl_listener *listener, void *data)
 	xwayland_update_workarea(server);
 }
 
+void
+xwayland_create_window_type_rules(void)
+{
+	// Desktop window type
+	struct window_rule *rule = znew(*rule);
+	rule->window_type = NET_WM_WINDOW_TYPE_DESKTOP;
+	rule->server_decoration = LAB_PROP_FALSE;
+	rule->skip_taskbar = LAB_PROP_TRUE;
+	rule->skip_window_switcher = LAB_PROP_TRUE;
+	rule->ignore_focus_request = LAB_PROP_TRUE;
+	rule->fixed_position = LAB_PROP_TRUE;
+	struct action *action_on_bottom = action_create("ToggleAlwaysOnBottom");
+	struct action *action_move_to = action_create("MoveTo");
+	action_arg_add_str(action_move_to, "x", "0");
+	action_arg_add_str(action_move_to, "y", "0");
```

Comment:
These should be `action_arg_add_int()` but it doesn't really matter that much as they default to `0` anyway. So these could also be removed.